### PR TITLE
SLE-603 Update okhttp and use it as an OSGi bundle

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: target-platforms/commons.target [hunk 1]
```diff
@@ -1,42 +1,70 @@
...
+				  <type>jar</type>
+			  </dependency>
+		  </dependencies>
+		  <instructions><![CDATA[
+Bundle-Name:           OkHttp
+version:               ${version_cleanup;${mvnVersion}}
+Bundle-SymbolicName:   com.squareup.okhttp3
+Bundle-Version:        ${version}
+Export-Package:        okhttp3;version="${version}";-noimport:=true
+Import-Package:        okio;version="3.0.0",kotlin.*;version="1.6.20"
```
Comment (damien-urruty-sonarsource) [inline target-platforms/commons.target:56]: Why is this needed ?

**Review by damien-urruty-sonarsource — APPROVED** [review]
One question, else LGTM

File: target-platforms/commons.target (hunk 1 above)
Comment (henryju) [reply target-platforms/commons.target:56]: Because okhttp is not an OSGi library (while okio is...):
image

Comment (henryju) [reply target-platforms/commons.target:56]: Full OSGi support is coming in version 5: https://square.github.io/okhttp/changelogs/changelog/#version-500-alpha1
